app/services/create_nomis_visit.rb

```diff
@@ -0,0 +1,54 @@
+class CreateNomisVisit
+  ALREADY_PROCESSED = 'Duplicate post'.freeze
+
+  def initialize(visit)
```

Comment:
@StupidCodeFactory I know you suggested passing the parameters instead of the visit. I think it makes sense to keep it with the visit as it keeps the logic of how to persist a PVB visit to the API here, and not spreading the logic to the `Accept` class.

What do you think @doctorpod ?